doc/contributing/distribution.md

```diff
@@ -0,0 +1,24 @@
+# Node.js Distribution Policy
+
+This document describes some policies around what is and is not included in the
+Node.js distribution.
+
+## Inclusion
+
+Node.js includes some external projects that the Node.js team does not maintain.
+The fact of a project's inclusion should not imply anything about the project
+relative to its competitors; in some cases, a project was added when it had no
```

Comment:
This was to address @mcollina’s note about how some choices of what to include were made for historical reasons.

Reply:
Maybe it would help if we find a different word than project, and only the word "project" to refer to the Node.js project. Maybe "dependency" would work?

Reply:
The problem with “dependency” is that npm isn’t really a dependency of Node, nor is Corepack. They’re just additional pieces of software we happen to include as part of our distribution. Perhaps this?

```suggestion
The fact of a tool or library's inclusion should not imply anything about it
relative to its competitors; in some cases, something was added when it had no
```